lib/modules/@apostrophecms/utils/src/apos/always/http.js

```diff
@@ -148,13 +152,24 @@
       data = options.body;
     }
     xmlhttp.send(data);
-    xmlhttp.addEventListener("load", function() {
-      var responseHeader = this.getResponseHeader("Content-Type");
+    xmlhttp.addEventListener('load', function() {
+      if (options.busy) {
```

Comment:
I have an idea for a cleaner resolution. let's have the `busy` option take a value (a string), and emit that string as the payload of the event. For a contextual rich text save operation, it can be "save-in-context", and then the frontend team can worry about displaying it in an appropriate vue component. No need for us to hack on the DOM.

If `options.busy` is just `true`, we can pass `busy` as the payload of the event (the general case).